src/main/java/uk/gov/ons/ctp/response/collection/exercise/message/impl/CollectionExerciseEventInboundReceiver.java

```diff
@@ -45,7 +45,9 @@ public void handleEventMessage(final EventMessageDTO message) {
                 this.collectionExerciseService.transitionCollectionExercise(collexId,
                         CollectionExerciseDTO.CollectionExerciseEvent.GO_LIVE);
             } catch (CTPException e) {
-                log.error("Failed to set collection exerise to live state - {}", e);
+                // The most likely cause for this exception is because the collection exercise is already in the live
+                // state
+                log.warn("Failed to transition collection exercise to live state: {}", e.getMessage());
```

Comment:
What's the reason for surprising the stacktrace?

Reply:
suppressing**

Reply:
Yeah suppressing. Must have been surprised it was being suppressed 😄

Reply:
I was reducing the noise in the logs but now I look at this one again, this should probably be an error with a stack trace. The noisy error is when you set the go_live date to the past when the collection exercise isn't in the READY_FOR_LIVE state but this is a debug situation - it's not worth hiding possible real exceptions so I'll revert this one